utils/model.py

```diff
@@ -235,13 +236,16 @@ def splits(cls, config):
                 elif label == words[cls.LABEL_UNKNOWN]:
                     unknown_files.append(wav_name)
                     continue
-                bucket = int(hashlib.sha1(filename.encode()).hexdigest(), 16) % 100
-                if bucket < train_pct:
-                    tag = 0
-                elif bucket < train_pct + dev_pct:
+                hashname = re.sub(r"_nohash_.*$", "", filename)
+                max_no_wavs = 2**27 - 1
+                bucket = int(hashlib.sha1(hashname.encode()).hexdigest(), 16)
+                bucket = (bucket % (max_no_wavs + 1)) * (100. / max_no_wavs)
+                if bucket < dev_pct:
                     tag = 1
```

Comment:
Should we use an Enum for these tags to specify what they mean?

Reply:
Sure